go/registry/api/runtime.go

```diff
@@ -113,10 +111,6 @@ func (e *ExecutorParameters) ValidateBasic() error {
 		return fmt.Errorf("round timeout too small")
 	}
 
-	if e.MinPoolSize < e.GroupSize+e.GroupBackupSize {
-		return fmt.Errorf("minimum pool size too small")
```

Comment:
do these checks get reinstated elsewhere?

Reply:
No, we could add them back to catch nonsensical configs.